test/unit/api_spec.js

```diff
@@ -1192,7 +1193,8 @@ describe('api', function() {
   });
   describe('Multiple PDFJS instances', function() {
```

Comment:
Also while you're here, let's update the name to `Multiple getDocument instances` (which I overlooked when removing the global `PDFJS` object).